marblerun/workflows/add-service.md

```diff
@@ -45,6 +46,52 @@ The tool's output will look like the following.
 }
 ```
 
+#### Graphene
+
+To add an entry for your Graphene service, run the `graphene-sgx-get-token` tool on the `.sig` file you built in the previous step as follows. (`graphene-sgx-get-token` is installed with [Graphene](https://github.com/oscarlab/graphene/).)
+
+
+```bash
+graphene-sgx-get-token --sig hello.sig
+```
+
+The tool's output will look like the following.
```

Comment:
```suggestion
The tool's output is similar to the following.
```
Also check the other 2 mentions.
Sounds a bit cleaner in my opinion.
Taken from the [google documentation style guide](https://developers.google.com/style/code-syntax).